src/main/java/uk/gov/hmcts/cmc/claimstore/services/ccd/CoreCaseDataService.java

```diff
@@ -105,7 +105,7 @@ public CaseReference savePrePayment(String externalId, String authorisation) {
 
             StartEventResponse startEventResponse = startCreate(authorisation, eventRequestData, user.isSolicitor());
 
-            CaseDataContent caseDataContent = CaseDataContent.builder()
+            CaseDataContent caseData = CaseDataContent.builder()
```

Comment:
revert the changes in this file please, they are unnecessary and there's already lots of changes in this PR